fvm/environment/uuids.go

```diff
@@ -90,14 +174,17 @@ func (generator *uUIDGenerator) GenerateUUID() (uint64, error) {
 		return 0, fmt.Errorf("generate uuid failed: %w", err)
 	}
 
-	uuid, err := generator.getUUID()
+	generator.maybeInitializePartition()
+
+	value, err := generator.getUint64()
 	if err != nil {
 		return 0, fmt.Errorf("cannot generate UUID: %w", err)
 	}
 
-	err = generator.setUUID(uuid + 1)
+	err = generator.setUint56(value + 1)
 	if err != nil {
 		return 0, fmt.Errorf("cannot generate UUID: %w", err)
 	}
-	return uuid, nil
+
+	return (uint64(generator.partition) << 56) | value, nil
```

Comment:
```suggestion
	// lift the value with the partition of the transaction, so that transactions in different partition
	// will never have overlapped uuid, because the first byte of the uuid value (the partition) is always 
	// different
	return (uint64(generator.partition) << 56) | value, nil
```